static/src/javascripts/projects/common/modules/identity/upsell/upsell.js

```diff
@@ -37,13 +38,16 @@ const ConfirmEmailThankYou = (
 );
 
 const Optouts = (
-    <Block
-        sideBySideBackwards
-        title="One more thing..."
-        subtitle="These are your privacy settings. You’re in full control of them.">
+    <Block sideBySideBackwards title="Your communication preferences">
         <LegalTextBlock>
-            You can also change these settings any time by visiting our Emails &
-            marketing section of your account.
+            You can change these preferences at any time via the&nbsp;
+            <a
+                data-link-name="upsell-optout-preferences-link"
+                className="u-underline identity-upsell-consent-card__link"
```

Comment:
there's some css in the identity project (`.identity-wrapper a.u-underline`) that makes these links black instead of the old blue, could it be applied to this page too? You can use `.identity-upsell-layout` as a wrapper selector